web/gui/dashboard_info.js

```diff
@@ -3654,4 +3654,88 @@ netdataDashboard.context = {
             },
         ],
     },
+
+    // ------------------------------------------------------------------------
+    // Nvidia-smi
+
+    'nvidia_smi.fan_speed': {
+        heads: [
+            function (os, id) {
+                void (os);
+                return '<div data-netdata="' + id + '"'
+                    + ' data-dimensions="speed"'
+                    + ' data-chart-library="easypiechart"'
+                    + ' data-title="Fan Speed"'
+                    + ' data-units="percentage"'
+                    + ' data-gauge-adjust="width"'
+                    + ' data-width="12%"'
+                    + ' data-before="0"'
+                    + ' data-after="-CHART_DURATION"'
+                    + ' data-points="CHART_DURATION"'
+                    + ' data-colors="' + NETDATA.colors[4] + '"'
+                    + ' data-decimal-digits="2"'
+                    + ' role="application"></div>';
+            }
+        ]
+    },
+    'nvidia_smi.temperature': {
+        heads: [
+            function (os, id) {
+                void (os);
+                return '<div data-netdata="' + id + '"'
+                    + ' data-dimensions="temp"'
+                    + ' data-chart-library="easypiechart"'
+                    + ' data-title="Temperature"'
+                    + ' data-units="celsius"'
+                    + ' data-gauge-adjust="width"'
+                    + ' data-width="12%"'
+                    + ' data-before="0"'
+                    + ' data-after="-CHART_DURATION"'
+                    + ' data-points="CHART_DURATION"'
+                    + ' data-colors="' + NETDATA.colors[3] + '"'
+                    + ' data-decimal-digits="2"'
+                    + ' role="application"></div>';
+            }
+        ]
+    },
+    'nvidia_smi.memory_allocated': {
+        heads: [
+            function (os, id) {
+                void (os);
+                return '<div data-netdata="' + id + '"'
+                    + ' data-dimensions="used"'
+                    + ' data-chart-library="easypiechart"'
+                    + ' data-title="Used Memory"'
+                    + ' data-units="MiB"'
+                    + ' data-gauge-adjust="width"'
+                    + ' data-width="12%"'
+                    + ' data-before="0"'
+                    + ' data-after="-CHART_DURATION"'
+                    + ' data-points="CHART_DURATION"'
+                    + ' data-colors="' + NETDATA.colors[4] + '"'
+                    + ' data-decimal-digits="2"'
+                    + ' role="application"></div>';
+            }
+        ]
+    },
+    'nvidia_smi.power': {
+        heads: [
+            function (os, id) {
+                void (os);
+                return '<div data-netdata="' + id + '"'
+                    + ' data-dimensions="power"'
+                    + ' data-chart-library="easypiechart"'
+                    + ' data-title="Power Utilization"'
+                    + ' data-units="watts"'
```

Comment:
I think Watts should be with `W` in uppercase, because I never saw this unit in lower case. :thinking:

Reply:
You mean in the `dashboard_info.js`?

Reply:
Yes, this lowercase does not make sense.

Reply:
Did you check it? 😄 

```cmd
0 gui (nvidia-smi_web_info)$ grep "data-units" dashboard_info.js
                    + ' data-units="percentage %"'
                    + ' data-units="percentage %"'
                    + ' data-units="percentage %"'
                    + ' data-units=""'
                    + ' data-units="%"'
                        + ' data-units="%"'
                    + ' data-units="%"'
                        + ' data-units="MB"'
                    + ' data-units="KB/s"'
                    + ' data-units="KB/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="milliseconds"'
                    + ' data-units="requests/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="milliseconds"'
                    + ' data-units="requests/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="requests/s"'
                    + ' data-units="milliseconds"'
                    + ' data-units="clients"'
                    + ' data-units="processes"'
                    + ' data-units="messages"'
                    + ' data-units="messages/s"'
                    + ' data-units="messages/s"'
                    + ' data-units="percentage"'
                    + ' data-units="messages/s"'
                    + ' data-units="messages/s"'
                    + ' data-units="messages/s"'
                    + ' data-units="subscriptions"'
                    + ' data-units="messages"'
                    + ' data-units="messages/s"'
                    + ' data-units="messages/s"'
                    + ' data-units="messages/s"'
                    + ' data-units="subscriptions"'
                    + ' data-units="messages"'
                    + ' data-units="percentage"'
                    + ' data-units="celsius"'
                    + ' data-units="MiB"'
                    + ' data-units="watts"'
```

Reply:
it is ok to have to have it lowercase https://en.wikipedia.org/wiki/SI_derived_unit